src/knock.ts

```diff
@@ -59,6 +73,28 @@ class Knock {
     return this.workflows.trigger(workflowKey, properties);
   }
 
+  signToken(userId: string, options?: SignTokenOptions) {
+    if (!this.signingKey) throw new NoSigningKeyProvidedException();
+
+  // JWT NumericDates specified in seconds:
+  const currentTime = Math.floor(Date.now() / 1000);
+  
+  // Default to 1 hour from now
+  const expireInSeconds = options?.expiresInSeconds ?? 60 * 60;
+
+  return jwt.sign(
```

Comment:
The indentation in this function looks off to me – can we run prettier on this before we're done here